scripts/compare-versions/src/main.rs

```diff
@@ -0,0 +1,228 @@
+use anyhow::Result;
```

Comment:
Perhaps  we could at some crate root documentation here (e.g. `//!`) that describes the purpose of this script. E.g. that the purpose is to fetch the recent workflow runs to find new versions and then outputs all combinations of untested versions to `stdout`